src/app/containers/Ad/Canonical/index.jsx

```diff
@@ -1,6 +1,14 @@
-const CanonicalAd = () => {
-  // To be implemented in the next feature ticket: https://github.com/bbc/simorgh/issues/6182
-  return null;
-};
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import AdSlot from './AdSlot';
+
+const CanonicalAd = () => (
+  <>
+    <Helmet>
+      <script src="https://gn-web-assets.api.bbc.com/ngas/test/dotcom-bootstrap.js" />
+    </Helmet>
+    <AdSlot uniqueId="leaderboard" />
```

Comment:
This should be a prop passed into CanonicalAd, since it'll be used in multiple places.

```suggestion
    <AdSlot uniqueId={uniqueId} />
```

Reply:
I can see this file being renamed to `FrontPageLeaderboardAd.jsx` or something when we have multiple, since it's more the setup for the leaderboard than a reusable component. If we have to pass the `uniqueId` prop into it I'd say that it would be better to move this code into `AdSlot.jsx`. If Helmet avoids script duplication for us then I'd say that's a good approach.

Reply:
Is there any reason why this can't be re-used for both leaderboard and mpus across all page types that will need display ads?

Reply:
Might be best to raise it as a separate issue since the same consideration needs to be made for AmpAd.

Reply:
I had imagined that the top level ad component would take a prop for whether we wanted a leaderboard or an mpu, which could then be passed down into either the canonical or amp ad container - probably best we come to an agreement on this!

Reply:
Yeah something like this would make sense

1. Roll `Canonical/index.jsx` and `AdSlot` into one (ensure script loading should be here)
2. Allow passing of `slotType` (leaderboard, mpu, etc) to both Canonical & Amp (GNL are defaulting to what we call leaderboard at the moment. I believe the names will be for us to decide, but I'll clarify at todays meeting).
3. AmpAd also takes service prop, which Canonical doesn't require since this is done for us.

Reply:
Cool - shall we get this up on test as is, and then ensure we capture doing the work to pass in the slot type?